libs/s25main/ingameWindows/IngameWindow.h

```diff
@@ -71,7 +71,7 @@ class IngameWindow : public Window
     DrawPoint GetRightBottomBoundary();
 
     /// Set the position for the window after adjusting newPos so the window is in the visible area
-    void SetPos(DrawPoint newPos);
+    void SetPos(DrawPoint newPos, bool saveRestorePos = true);
```

Comment:
How do you feel about this publicly exposed parameter? I was considering hiding it in a private function. I don't think it's useful outside of `IngameWindow`.

Reply:
Not strongly either way as I'm not sure if it is worth adding an extra private function for this. Might even lead to possible issues.

Reply:
I'm worried about potential issues too. But because an internal feature is now accidentally usable from the outside. :man_shrugging:

Reply:
Guess we'll find out.